src/backend/tests/integration/components/output_parsers/test_csv_output_parser.py

```diff
@@ -0,0 +1,44 @@
+from langflow.components.models.OpenAIModel import OpenAIModelComponent
+from langflow.components.output_parsers.CSVOutputParser import CSVOutputParserComponent
+from langflow.components.prompts.Prompt import PromptComponent
+from langflow.schema.message import Message
+from langflow.template.field.base import Input
+from langflow.template.template.base import Template
+from tests.integration.utils import ComponentInputHandle, run_single_component
+
+import pytest
+
+
+@pytest.mark.asyncio
+@pytest.mark.api_key_required
+@pytest.mark.skip(reason="template field not validating correctly")
+async def test_csv_output_parser_openai():
```

Comment:
@nicoloboschi Does the below component setup look correct to you? It's failing to validate the `template` component:
```
src/backend/tests/integration/utils.py:166: in run_single_component
    component_id = _add_component(clazz, inputs)
src/backend/tests/integration/utils.py:162: in _add_component
    handle_component_id = _add_component(handle.clazz, handle.inputs)
src/backend/tests/integration/utils.py:158: in _add_component
    component_id = graph.add_component(component)
src/backend/base/langflow/graph/graph/base.py:213: in add_component
    frontend_node = component.to_frontend_node()
src/backend/base/langflow/custom/custom_component/component.py:515: in to_frontend_node
    frontend_node = ComponentFrontendNode.from_dict(frontend_node_dict)
src/backend/base/langflow/template/frontend_node/base.py:94: in from_dict
    data["template"] = Template.from_dict(data["template"])
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

cls = <class 'langflow.template.template.base.Template'>, data = {'type_name': 'Component'}

    @classmethod
    def from_dict(cls, data: dict) -> "Template":
        for key, value in data.copy().items():
            if key == "_type":
                data["type_name"] = value
                del data[key]
            else:
                value["name"] = key
                if "fields" not in data:
                    data["fields"] = []
                input_type = value.pop("_input_type", None)
                if input_type:
                    try:
                        _input = instantiate_input(input_type, value)
                    except Exception as e:
                        raise ValueError(f"Error instantiating input {input_type}: {e}")
                else:
                    _input = Input(**value)
    
                data["fields"].append(_input)
>       return cls(**data)
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for Template
E       fields
E         Field required [type=missing, input_value={'type_name': 'Component'}, input_type=dict]
E           For further information visit https://errors.pydantic.dev/2.9/v/missing

src/backend/base/langflow/template/template/base.py:59: ValidationError
```